com.unity.render-pipelines.high-definition/Runtime/Lighting/Light/HDAdditionalLightData.cs

```diff
@@ -2678,6 +2682,18 @@ void LateUpdate()
                 timelineWorkaround.oldDisplayAreaLightEmissiveMesh = displayAreaLightEmissiveMesh;
                 timelineWorkaround.oldLightColorTemperature = legacyLight.colorTemperature;
             }
+
+            if (timelineWorkaround.renderEntityDataVersion != m_CurrentRenderEntityVersion)
```

Comment:
It looks to me like none of this will work properly in player builds because of https://github.com/Unity-Technologies/Graphics/pull/6626/files#diff-ce5da627f91b89bc1411c29b4bba111e4197513b924bbb3fa614ed87f2f74c34R2576-R2579